scripts/qa_crowbarsetup.sh

```diff
@@ -823,53 +823,53 @@ function onadmin_set_source_variables()
         develcloud3)
             CLOUDDISTPATH=/ibs/Devel:/Cloud:/3/images/iso
             [ -n "$TESTHEAD" ] && CLOUDDISTPATH=/ibs/Devel:/Cloud:/3:/Staging/images/iso
-            CLOUDDISTISO="S*-CLOUD*Media1.iso"
+            CLOUDSLE11DISTISO="S*-CLOUD*Media1.iso"
             CLOUDLOCALREPOS="SUSE-Cloud-3-devel"
         ;;
         develcloud4)
             CLOUDDISTPATH=/ibs/Devel:/Cloud:/4/images/iso
             [ -n "$TESTHEAD" ] && CLOUDDISTPATH=/ibs/Devel:/Cloud:/4:/Staging/images/iso
-            CLOUDDISTISO="S*-CLOUD*Media1.iso"
+            CLOUDSLE11DISTISO="S*-CLOUD*Media1.iso"
             CLOUDLOCALREPOS="SUSE-Cloud-4-devel"
         ;;
         develcloud5)
             CLOUDDISTPATH=/ibs/Devel:/Cloud:/5/images/iso
             [ -n "$TESTHEAD" ] && CLOUDDISTPATH=/ibs/Devel:/Cloud:/5:/Staging/images/iso
             CLOUDCOMPUTEPATH=$CLOUDDISTPATH
-            CLOUDDISTISO="SUSE-CLOUD*Media1.iso"
-            CLOUDCOMPUTEISO="SUSE-SLE12-CLOUD-5-COMPUTE-x86_64*Media1.iso"
+            CLOUDSLE11DISTISO="SUSE-CLOUD*Media1.iso"
+            CLOUDSLE12DISTISO="SUSE-SLE12-CLOUD-5-COMPUTE-x86_64*Media1.iso"
             CLOUDLOCALREPOS="SUSE-Cloud-5-devel"
         ;;
         develcloud6)
             CLOUDDISTPATH=/ibs/Devel:/Cloud:/6/images/iso
             [ -n "$TESTHEAD" ] && CLOUDDISTPATH=/ibs/Devel:/Cloud:/6:/Staging/images/iso
             CLOUDCOMPUTEPATH=$CLOUDDISTPATH
-            CLOUDDISTISO="SUSE-OPENSTACK-CLOUD-SLE11-6-$arch*Media1.iso"
-            CLOUDCOMPUTEISO="SUSE-SLE12-CLOUD-6-COMPUTE-x86_64*Media1.iso" # FIXME: not existing
+            CLOUDSLE11DISTISO="SUSE-OPENSTACK-CLOUD-SLE11-6-$arch*Media1.iso"
```

Comment:
@toabctl it's temporary; the current plan is to slowly phase out the SLE11 bits. But until it's done, we need a SLE11 ISO.